src/multiinstances/internal/ipc/ipc.h

```diff
@@ -44,6 +44,13 @@ static const QString IPC_WHOIS("ipc_whois");
 static const QString IPC_METAINFO("ipc_metainfo");
 static const QString IPC_PING("ipc_ping");
 
+enum class Code {
+    Undefined = -1,
+    Success = 0,
+    Timeout,
+    AllRecevied
```

Comment:
it's a bit confusing, let's say I've sent a message via syncBroadcast, so which one should I expect as the result - Success or AllReceived (btw there is typo here). I mean they look similiar to me

Reply:
> it's a bit confusing, let's say I've sent a message via syncBroadcast, so which one should I expect as the result - Success or AllReceived (btw there is typo here). I mean they look similiar to me

The idea here is that we do request for all, and put callback, that checking responses. If someone responses success for us, we break wait others, and will be returned Success.
If no one answers us with success, and there is no one else to wait (everyone answered), then the result will be returned AllRecevied (maybe better to rename AllAnswered)

Like, 
1. Has anyone opened this one score?
not
not
yes -> Success for us 
-- break --
not

2. Has anyone opened this one score?
not
not
not  
not
-> AllReceived

Reply:
Yeah, I see now. Perhaps it would make sense to have the "retval" semantic, but I'm not quite sure how to put it in here

Reply:
renamed to `AllAnswered`